docs/features/sidebar-navigation.md

```
@@ -25,12 +25,21 @@ The switcher loads family members through `useFamilyMembers({ enabled: canSwitch
 
 ### Mobile: `MobileBottomNav` (hidden on desktop via `md:hidden`)
 
-A fixed bottom bar with the Picsou logo centered and 2 nav items on each side:
+A fixed bottom bar with five evenly spaced nav items:
 
 ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)